packages/mask/src/components/shared/ApplicationSettingPluginSwitch.tsx

```diff
@@ -93,7 +93,7 @@ export const ApplicationSettingPluginSwitch = memo(({ focusPluginID }: Props) =>
     const availablePlugins = useMemo(() => {
         return snsAdaptorPlugins
             .flatMap(({ ID, ApplicationEntries: entries }) => (entries ?? []).map((entry) => ({ entry, pluginID: ID })))
-            .filter((x) => x.entry.category === 'dapp')
+            .filter((x) => x.entry.category === 'dapp' && x.pluginID !== PluginID.Trader)
```

Comment:
Is this the proper way? @Jack-Works

Reply:
Maybe use a new category type is a proper manner.

Reply:
+1. you can add a new property to indicate if it should be displayed in the switch